ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -283,6 +284,19 @@ func (r *RayClusterReconciler) rayClusterReconcile(ctx context.Context, request
 							"We do not need to requeue the RayCluster CR anymore.", instance.Name))
 					return ctrl.Result{}, nil
 				}
+				// redisCleanupJob.Spec.ActiveDeadlineSeconds is set by the GCS_FT_REDIS_CLEANUP_DEADLINE_SECONDS env
```

Comment:
Can we merge this `if` statement and the `if redisCleanupJob.Status.Succeeded > 0` statement?

```
if utils.IsJobFinished(&redisCleanupJob) {
  isBestEffort := redisCleanupJob.Spec.ActiveDeadlineSeconds != nil
  if (Job is complete) {
    print some logs
  } else {
    print some logs
    maybe file a k8s event
  }
  remove finalizer
}
```

Reply:
Done.